sql/lens_ids.sql

```diff
@@ -0,0 +1,6 @@
+-- get lens article ids (used in validation)
+SELECT
+  DISTINCT(lens_id) as id
+FROM
+  lens.scholarly
+WHERE (publication_type IS NULL) OR (NOT (publication_type IN ("dataset", "editorial", "letter", "news", "review")))
```

Comment:
Please sanity check me here... there are 25 of these, and 73 "publication_supplementary_type"s!